examples/common_components/protocol_examples_common/Kconfig.projbuild

```diff
@@ -419,6 +432,44 @@ menu "Example Connection Configuration"
 
         endchoice
 
+        config LWIP_IPV6_AUTOCONFIG
+            bool "Enable IPV6 stateless address autoconfiguration (SLAAC)"
+            depends on LWIP_IPV6
+            default y
+            help
+                Enabling this option allows the devices to IPV6 stateless address autoconfiguration (SLAAC).
+
+                See RFC 4862.
+
```

Comment:
> To change the default values please use sdkconfig.defaults in example.

Cool, thanks for the feedback. I didn't know you could do that. Yes, all I want to do is set some reasonable defaults to have both IPv4 and IPv6 fully working (where available). I will update.

Reply:
@sgryphon, you can utilize the `idf.py save-defconfig` command to save the current configuration into a 'sdkconfig.defaults' file, which will be utilized thereafter as default configuration.

Reply:
I have moved the config defaults into `sdkconfig.defaults` instead of repeating them in the example Kconfig.

However this fix still needs to wait for the changes in LWIP https://github.com/espressif/esp-lwip/pull/66

Reply:
I also made a similar change to my other PR for the HTTP example, that streamlines the common protocol example code to also work smoothly across any network: https://github.com/espressif/esp-idf/pull/13250